test/lib/dependency-versions-test.ts

```diff
@@ -418,41 +403,38 @@ describe('Utils | dependency-versions', function () {
       ]);
 
       // Existing newline at end of file should be maintained.
-      ok(
-        !packageJson1Contents.endsWith('\n'),
-        'package1 should not end in newline'
-      );
-      ok(packageJson2Contents.endsWith('\n'), 'package2 should end in newline');
+      expect(!packageJson1Contents.endsWith('\n')).toBeTruthy();
+      expect(packageJson2Contents.endsWith('\n')).toBeTruthy();
```

Comment:
I think there's a more-specific we can use for string matching which would provide better output?

https://jestjs.io/docs/expect#expectstringmatchingstring--regexp

Reply:
Will this work? `endsWith()` returns a boolean and not a string.

Reply:
You have to actually change it to use a regexp instead of `endsWith()`.

```
expect(...).not.toMatch('\n$')
expect(...).toMatch('\n$')
```